api/src/docker/openscad/runScad.js

```diff
@@ -14,23 +12,23 @@ module.exports.runScad = async ({
     } = {},
   } = {}, // TODO add view settings
 } = {}) => {
-  const tempFile = await makeFile(file)
+  const tempFile = await makeFile(file, '.scad', nanoid)
   const { x: rx, y: ry, z: rz } = rotation
   const { x: px, y: py, z: pz } = position
   const cameraArg = `--camera=${px},${py},${pz},${rx},${ry},${rz},${dist}`
   const command = `xvfb-run --auto-servernum --server-args "-screen 0 1024x768x24" openscad -o /tmp/${tempFile}/output.png ${cameraArg} --imgsize=${x},${y} --colorscheme DeepOcean /tmp/${tempFile}/main.scad`
   console.log('command', command)
 
   try {
-    const result = await runCommand(command, 10000)
+    const result = await runCommand(command, 15000)
```

Comment:
You've managed to make this backend code really readable even without me having run it before, cheers.